tests/functional/adapter/concurrent_transaction/test_concurrent_transaction.py

```diff
@@ -0,0 +1,120 @@
+import time
+
+import pytest
+import threading
+
+from dbt.tests.util import run_dbt
+from dbt.tests.adapter.simple_seed.test_seed import SeedConfigBase
+from tests.functional.adapter.concurrent_transaction.fixtures import *
+
+
+class BaseConcurrentTransaction:
+    @pytest.fixture(scope="function", autouse=True)
+    def setUp(self, project):
```

Comment:
do we want this to be the setup or should re rename back to `reset` like the integration test

Reply:
@mikealfare suggested changing this earlier. Went back and forth on it. I don't have a preference. I'll get it to reset() for now to be on par with the previous integration.

Reply:
i'd have to try to find the context but I think setUp has some special context for triggering namewise as well so might be kicking stuff off when we don't want to